_docs/sdk/sdk-installation-guide.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Smartlook SDK Installation Guide"
+subtitle: "A handy guide to find the best installation details for your project."
+description: "Installation guide for Smartlook SDK."
+---
+
+# Smartlook SDK Installation Guide
+
+
+To start please select your mobile OS: 
+
+
+|    Android   | iOS |   
+|:--------------:|:--------:|
+|  [![Android Icon](https://img.icons8.com/color/96/000000/android-os.png)](../sdk/sdk-installation-guide.md#android-native-installation--supported-frameworks) | [![iOS Icon](https://img.icons8.com/ios/99/000000/ios-logo.png)](../sdk/sdk-installation-guide.md#ios-native-installation--supported-frameworks)  
+|  [Android & Supported Frameworks](../sdk/sdk-installation-guide.md#android-native-installation--supported-frameworks) | [iOS & Supported Frameworks](../sdk/sdk-installation-guide.md#ios-native-installation--supported-frameworks) | 
```

Comment:
Remove `../sdk/sdk-installation-guide.md` and leave just the anchor for both the links.